cmd/cluster/aws/create.go

```diff
@@ -156,6 +156,19 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 		tags = append(tags, hyperv1.AWSResourceTag{Key: k, Value: v})
 	}
 
+	var instanceType string
+	if opts.AWSPlatform.InstanceType != "" {
+		instanceType = opts.AWSPlatform.InstanceType
+	} else {
+		// Aligning with AWS IPI instance type defaults
+		switch opts.Arch {
```

Comment:
these strings are used in both files should all come from constants, ideally even reuse the ones in NodePool controller.

Reply:
Ok I moved those constants from the NodePool controller to nodepool_types.go in the api so that the same constants are used in both here and in the NodePool controller.